CIP-0010/registry.json

```diff
@@ -41,7 +41,7 @@
   },
   {
     "transaction_metadatum_label": 839,
-    "description": "Agora - Proposal creation metadata"    
```

Comment:
Should I undo this?

Reply:
yes; please take it out: it's better to keep other lines the same to ensure each label has a unique PR associated with it.  If white space is a categorical problem, editors can submit a PR to clean it up or parse it out with some GitHub CI that is running on this file.

Reply:
Yeah that make sense. Someone could just do a single commit to remove all the unnecessary white spaces and add the commit to `.git-blame-ignore-revs` file